website/static/js/profile.js

```diff
@@ -924,6 +929,11 @@ ListViewModel.prototype.unserialize = function(data) {
         self.editable(false);
     }
     self.contents(ko.utils.arrayMap(data.contents || [], function (each) {
+        for (var attr in each) {
```

Comment:
Nice! One thing, it's preferable to use for-in loops only for objects with string keys not arrays. There are other drawbacks of concern. Please check some good practices [here](https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/).